qubes/config.py

```diff
@@ -44,6 +44,8 @@
     # qubes_icon_dir is obsolete
     # use QIcon.fromTheme() where applicable
     'qubes_icon_dir': '/usr/share/icons/hicolor/128x128/devices',
+
+    'dom0_services_dir': '/var/lib/qubes-services',
```

Comment:
This should be `/var/run/qubes-service`.

Reply:
Yes indeed.